resources/public/api/conf/1.0/examples/register/register-max-post.json

```diff
@@ -6,34 +6,36 @@
   "birthDateVerification": "VERIFIED",
   "officeNumber": "1234",
   "contactNumber": "1234567890",
-  "country": 1,
-  "name": {
+  "applicantNames": {
+    "nameType":"REGISTERED",
     "title": "MR",
     "forename": "AForename",
     "secondForename": "NotSure",
     "surname": "ASurname",
     "startDate": "10-10-2020",
     "endDate": "10-10-2020"
   },
-  "historicNames": [
+  "applicantHistoricNames": [
     {
+      "nameType":"REGISTERED",
       "title": "MRS",
       "forename": "AForename",
-      "secondForename": "NotSure",
+      "middleName": "NotSure",
```

Comment:
This needs changing back to 'secondForename'

Reply:
done